#!/usr/bin/env python3
PACKAGE = "prius_control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("Kp", double_t, 0, "A Kp parameter", 0.0, 0.0, 20.0)
gen.add("Ki", double_t, 0, "A Ki parameter", 0.0, 0.0, 20.0)
gen.add("Kd", double_t, 0, "A Kd parameter", 0.0, 0.0, 20.0)
gen.add("Setpoint_first", double_t, 0, "First setpoint parameter", 12.0, 0.0, 160.0)
gen.add("Setpoint_second", double_t, 0, "Second setpoint parameter", 15.0, 0.0, 160.0)

exit(gen.generate(PACKAGE, "prius_velocity_pid_node", "pid"))
